server/core/peer.go

```diff
@@ -72,3 +72,66 @@ func CountInJointState(peers ...*metapb.Peer) int {
 	}
 	return count
 }
+
+// PeerInfo provides peer information
+type PeerInfo struct {
+	*metapb.Peer
+	writtenBytes uint64
+	writtenKeys  uint64
+	readBytes    uint64
+	readKeys     uint64
```

Comment:
would you like to replace them with []float64?

Reply:
We will change it with regionInfo together in future.